source/renderer/zettlr-body.js

```diff
@@ -151,17 +149,24 @@ class ZettlrBody {
    * necessary adjustments.
    */
   configChange () {
-    let newTheme = global.config.get('display.theme')
-    // Check if we really need to replace the style to prevent ugly flickering
-    // when replacing the same theme with the same.
-    if (this._currentTheme === newTheme) return
-
     // On config change, change the theme according to the settings
-    let href = $('link#theme-css').attr('href')
-    href = href.replace(/bielefeld|berlin|frankfurt|karl-marx-stadt|bordeaux/, newTheme)
-    $('link#theme-css').attr('href', href)
-    this._renderer.getEditor().refresh()
-    this._currentTheme = newTheme
+    let newThemeName = global.config.get('display.theme')
+
+    const themes = {
```

Comment:
I am sure I do not fully understand this code, but it looks like "bad practice".

I am missing the place where the LESS files are converted to CSS. And if it might introduce problems if unuse is used more often than the require, we should not do this.

But just for clarification:

1. Webpack compiles the templates to CSS and stored them as a JS-module within the final renderer bundle
2. "require" just puts the theme in the variables
3. Webpack immediately adds the CSS as links into the head of the file (that is: all five of them)
4. You remove those that are not correct

This really does not work good. Rather, I would like to propose the following two alternatives:

1. Put ALL themes into one big CSS module (if everything is within one file either way, we can completely throw overboard the idea of separating the file in the first place, and only keep them separated in the source), but namespace them (e.g. `body.theme-berlin` and `body.theme-berlin.dark`) and deal with this using classes
2. Remove that fancy Webpack magic (which adds a lot of intransparency to the build process either way and is one of the major reasons I really do not like working with any JavaScript bundler …)

Either way, the logic here is completely opaque to the non-initiated and we need to comment what is exactly happening here and why that is so.

Reply:
Tobi: store current theme + unuse it

Reply:
Done